src/main/java/Crystal.java

```diff
@@ -0,0 +1,298 @@
+import java.util.Scanner;
```

Comment:
nit: You could arrange java-provided libraries imports together and separate self-defined packages imports.